### PR TITLE
tail: add a --force flag to bypass the DO prompt

### PR DESCRIPTION
**Pull request by elithrar**
This PR adds a `--force` flag to `wrangler tail` to bypass the confirmation prompt for WebSocket/DO.

* It will still print the warning text
* Adds a changeset

cc @lrapoport-cf

### CODE REVIEW

**Comment by jspspike** [comment]
This looks good to me, do you mind opening a PR in cloudflare-docs to update [this page](https://github.com/cloudflare/cloudflare-docs/blob/production/content/workers/wrangler/commands.md) with the new option?

**Comment by elithrar** [comment]
@jspspike - already done / already mentioned in both PRs: https://github.com/cloudflare/cloudflare-docs/pull/11827

**Review by jspspike — APPROVED** [review]
Sorry didn't see the docs PR in the activity, could maybe use a test with the option. Otherwise looks good to me

**Comment by lrapoport-cf** [comment]
hey @elithrar :) we discussed this as a team a bit more and agreed it would be good to have a test --  could you please add one? apologies we didn't catch it on the initial review!

File: .changeset/rotten-keys-occur.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
```
Comment (petebacondarwin) [inline .changeset/rotten-keys-occur.md:2]: Minor I think - this is a small feature not a bug fix.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Please can you add a test for this functionality.
There is already one for the interactive behaviour: https://github.com/cloudflare/workers-sdk/blob/93833f0418443232bb29daf46559c8e1db754dde/packages/wrangler/src/__tests__/tail.test.ts#L59

**Pull request closed without merging**